src/ProofOfConcepts/ConnectorMessageDrawer/DrawerDescList/ConnectorDrawer.css

```diff
@@ -0,0 +1,31 @@
+.connector-drawer__header-text {
+  margin-bottom: 0 !important;
+}
+.connector-drawer__header-title {
+  margin-top: 0 !important;
+}
+.connector-drawer__tab-content {
+  padding-top: var(--pf-global--spacer--lg);
+  padding-bottom: var(--pf-global--spacer--lg);
+}
+
+.pf-u-icon-color-green {
```

Comment:
If you use the `@patternfly/react-icons` components, you can use the `color` property to set this color, which I think would be better than doing this via CSS.  [Like this](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/ConnectorTypeCard/ConnectorTypeCard.tsx#L58) for example.  The constant I'm using there is from `@patternfly/react-tokens` which contains similar variables that [you can import](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/ConnectorTypeCard/ConnectorTypeCard.tsx#L22) and use.

Reply:
For what is worth, when migrating all the Kafka components to the new rhosak-ui repo I removed _all_ CSS files and replaced them all with PF utility classes where needed.

Reply:
Yep, totally agree.  For what we're working with it's really better to avoid needing CSS to lay out components, it's ideally handled by a combination of the components, their properties and the [utility classes](https://www.patternfly.org/v4/utilities/alignment).  And then when all that fails and you need that hammer :hammer:, you pull out the CSS :smile:

Reply:
I went thought the CSS rules and removed anything that was not relevant for the pr for now,  the only thing left was a CSS rule to make text grey